mock/py/mockbuild/uid.py

```diff
@@ -23,6 +47,7 @@ def __init__(self, unprivUid=-1, unprivGid=-1):
         self.unprivGid = unprivGid
         self.unprivEnviron = dict(os.environ)
         self.unprivEnviron['HOME'] = pwd.getpwuid(unprivUid).pw_dir
+        self.mockgid = grp.getgrnam('mock').gr_gid
```

Comment:
This is now done in two places instead of one.

Reply:
Then why not to keep the argument in `setup_uid_manager` (previous approach)?  It sounds
logical.

I get the problem you solve -- third-party app needs to realize what the mockgid is, but it is
fair to have another "library" method for that I think.

Reply:
> It sounds logical.

What I mean here: this is generic "setup_uid_manager()", not "setup_mock_uid_manager()"